### DISCUSSION TITLE
Aula Listas Assincronas: { "_isScalar": false, "source":

### DISCUSSION THREAD

**Discussion by ReneSaboiaMendesFilho** [balta-io/7187#62] [Perguntas e Respostas] [answered]
Estou na aula Listas Assincronas e não consegui mostrar os produtos, no HTML, inseri:  {{products$ | json}} e apareceu o seguinte:
{ "_isScalar": false, "source": { "_isScalar": false, "source": { "_isScalar": false, "source": { "_isScalar": false }, "operator": { "concurrent": 1 } }, "operator": {} }, "operator": {} }

![image](https://user-images.githubusercontent.com/85502566/145680379-d2d9ed04-2098-4fc2-9d11-7b44180a8093.png)
![image](https://user-images.githubusercontent.com/85502566/145680384-5b634517-89b6-4693-b2b9-f1bae6a46c6e.png)
![image](https://user-images.githubusercontent.com/85502566/145680395-a91a5645-5220-4cee-8ef9-b5b3c6dc552d.png)

**Comment by ReneSaboiaMendesFilho** [comment, marked as answer]
Resolvido, erro meu.

**Reply by andrebaltieri** [reply]
Perfeito 💜
